nano/qt/qt.cpp

```diff
@@ -74,14 +74,10 @@ balance_label (new QLabel),
 wallet (wallet_a)
 {
 	your_account_label->setStyleSheet ("font-weight: bold;");
-	std::string network = "Live";
-	if (wallet.node.network_params.network.is_beta_network ())
+	std::string network = wallet.node.network_params.network.get_current_network_as_string ();
+	if (!network.empty ())
```

Comment:
this is probably unnecessary (overly defensive) as network will always be populated, but its fine as well.